src/components/themed/WalletListSwipeable.tsx

```diff
@@ -66,7 +66,7 @@ function WalletListSwipeableComponent(props: Props) {
 
   const handleCreateWallet = useHandler(async (walletId: string, tokenId: EdgeTokenId) => {
     dispatch(selectWalletToken({ navigation, walletId, tokenId }))
-      .then(() => navigation.navigate('transactionList', { walletId, tokenId }))
+      .then(() => selectWalletToken({ navigation, walletId, tokenId }))
```

Comment:
Calling an action without dispatch doesn't do anything. This is equivalent to deleting the .then block completely. 

Instead, you can keep the transactionList navigation but make it conditional based on the return value of selectWalletToken which is false for unactivated wallets and true for every other case that successfully updates redux.

Reply:
Oops didn't notice the duplication here